nullaway/src/main/java/com/uber/nullaway/fixserialization/FixSerializationConfig.java

```diff
@@ -110,6 +129,12 @@ public FixSerializationConfig(String configFilePath) {
       throw new IllegalStateException(
           "Error in the fix serialization configuration, suggest flag must be enabled to activate enclosing method and class serialization.");
     }
+    methodParamProtectionTestEnabled =
+        XMLUtil.getValueFromAttribute(document, "/serialization/paramTest", "active", Boolean.class)
+            .orElse(false);
+    paramTestIndex =
+        XMLUtil.getValueFromAttribute(document, "/serialization/paramTest", "index", Integer.class)
+            .orElse(1000);
```

Comment:
The constant 1000 is a bit weird here, let's be consistent:
```suggestion
            .orElse(Integer.MAX_VALUE);
```